src/main/scala/firrtl/Compiler.scala

```diff
@@ -373,6 +373,9 @@ trait Compiler extends LazyLogging {
     */
   def transforms: Seq[Transform]
 
+  require(transforms.size >= 1,
+          s"Compiler transforms for '${this.getClass.getName}' was empty. A compiler must have at least ONE Transform!")
```

Comment:
Is it worth explicitly adding something like `IdentityTransform can be used as an identity/no-op transform.`?

Reply:
Good idea, I'll include this.